velox/functions/sparksql/CMakeLists.txt

```diff
@@ -18,6 +18,7 @@ add_library(
   Bitwise.cpp
   Comparisons.cpp
   DecimalArithmetic.cpp
+  DecimalCompareFunctions.cpp
```

Comment:
I wonder if it makes sense to rename this file to DecimalCompare.cpp for consistency with DecimalArithmetic.cpp and RegisterCompare.cpp.

Reply:
Makes sense. Renamed.